src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -566,6 +566,26 @@ public void RelativePath()
             }
         }
 
+        [Theory]
+        [InlineData(null, null)]
+        [InlineData("c:", null)]
+        [InlineData("c:\\", null)]
```

Comment:
>"c:\\" [](start = 20, length = 6)

Are these tests only run on Windows?

Reply:
I thought the logic in the method didn't depend on the current platform, but it looks like it does (directory separator is `\` or `/`). I'll make the test conditional.

---
In reply to: [378585981](https://github.com/dotnet/roslyn/pull/41063#discussion_r378585981) [](ancestors = 378585981)